### ISSUE TITLE
Missing version numbers for the dll

### ISSUE THREAD

**Issue by vnl** [KDAB/KDSoap#93]
Hello, 
I might be missing some option. Tried several builds now. I do get the required dll, and exe but the dll is missing the version number. 

![image](https://cloud.githubusercontent.com/assets/1355466/20934501/437e0346-bbd3-11e6-842b-0e57ff89c3d2.png)

Not sure if I am missing any options here. 

Options used to compile is:
configure.bat -shared -debug
and
configure.bat -shared -release

**Comment by dfaure** [comment]
This would be a qmake issue, I think.
If you look at the Qt DLLs themselves, are these versions set?

**Comment by vnl** [comment]
Yes the Qt DLL's have the versions. I hope you are asking about the actual Qt install (I use Qt 5.7.0). I did however find something within variables.pri file where If I comment line 37, #VERSION= then the version gets added. However the naming now changes to kdsoap11.dll
